src/annotator/components/test/NotebookModal-test.js

```diff
@@ -0,0 +1,135 @@
+import NotebookModal from '../NotebookModal';
+import { EventBus } from '../../util/emitter';
+import { mount } from 'enzyme';
+import { act } from 'preact/test-utils';
```

Comment:
Can you sort these with third party imports at the top please, then a gap, then local imports.

Reply:
No problem.